README.md

```diff
@@ -73,7 +73,7 @@ builds for released versions from [GitHub releases page](https://github.com/redh
         ```
         Or you can download it using following command:
         ```sh
-        sudo wget https://bintray.com/ocdev/ocdev-rpm-dev/rpm -O /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-dev.repo
+        sudo curl https://bintray.com/ocdev/ocdev-rpm-dev/rpm > /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-dev.repo
```

Comment:
`sudo curl https://bintray.com/ocdev/ocdev-rpm-dev/rpm -o /etc/yum.repos.d/bintray-ocdev-ocdev-rpm-dev.repo` will this be more ideal?